examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -309,6 +309,12 @@ void checkDigitalInputs(void)
 }
 
 // -----------------------------------------------------------------------------
+// function forward declarations
+void enableI2CPins();
```

Comment:
The Xtensa Compiler for the ESP8266 can't do without forward declarations if you want to call a function before it is defined. I havn't checked the dependencies. If there are no circular references  reordering the functions can avoid this.